internal/cli/atlas/deployments/scripts/start_mongot.sh

```diff
@@ -14,11 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-if [ ! -f "$KEYFILEPATH" ]
-then
-    echo "$KEYFILECONTENTS" > "$KEYFILEPATH"
-    chmod 400 "$KEYFILEPATH"
```

Comment:
is mongot not checking anymore the file permissions? (I think it was failing if permissions are too wide)

Reply:
it is done inside container

Reply:
https://github.com/10gen/mongot/pull/1802/files#diff-997d64eedc645601c81b86be6b25f569abfff63aaa3a49ff88975988a14065fbR177